cabot/cabotapp/views.py

```diff
@@ -219,7 +219,7 @@ class Meta:
             'username': forms.TextInput(attrs={
                 'style': 'width: 30%',
             }),
-            'password': forms.TextInput(attrs={
+            'password': forms.PasswordInput(attrs={
```

Comment:
I think my only concern is that doing it this way makes it a bit harder to debug. However yes let's go for it.